internal/controller/dispatcher.go

```diff
@@ -80,7 +89,9 @@ func (r *Dispatcher) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			// TODO: Multicluster. This is where we remove the placement that maps
```

Comment:
Update comment to remove TODO (or remove entirely if you think the code doesn't need a comment anymore)